package.json

```diff
@@ -4,7 +4,6 @@
   "description": "This project is a proposal for a react-based user interface for Drone.",
   "main": "webpack.config.js",
   "dependencies": {
-    "ansi_up": "^1.3.0",
     "baobab": "2.3.3",
```

Comment:
add the following and run `npm install` locally. This will fetch the dependency from the node package manager, which is a central repository of packages:

```diff
"dependencies": {
 -    "ansi_up": "^1.3.0",
 +    "aha": "0.1.0",
      "baobab": "2.3.3",
```